README.md

```diff
@@ -107,8 +107,16 @@ branch:
 2. `git merge main`
 3. `git push`
 
-> Note: As of now it's possible to push directly to production, but this will change after
-> https://github.com/api3dao/api3-dao-dashboard/issues/5 is resolved.
+#### Updating the name servers
+
+The primary way to access the DAO dashboard is through the `api3.eth` ENS name, which points directly to the IPFS hash.
+Then, the user can either connect to mainnet on their Metamask and visit `api3.eth/` (the recommended way), or they can visit `https://api3.eth.link/` (this is reported to be down frequently, see [this](https://blog.cloudflare.com/cloudflare-distributed-web-resolver/) for more information).
+We have also forwarded `https://dao.api3.org` to the IPFS hash (using the `dweb.link` gateway), but we do not recommend using this unless necessary.
```

Comment:
You probably want to use the http://api3dao.org/ we talked about in slack.

Reply:
I'll need to ask about it to others first